Compiler warning in matrix free task_info.cc

```
[869/989] Building CXX object source/matrix_free/CMakeFiles/obj_matrix_free_release.dir/task_info.cc.o
../source/matrix_free/task_info.cc:965:50: warning: suggest braces around initialization of subobject [-Wmissing-braces]
          const std::array<unsigned int, 3> next{category_hp, max_index, i};
                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~
                                                 {                        }
1 warning generated.
```

--- a/source/matrix_free/task_info.cc
+++ b/source/matrix_free/task_info.cc
@@ -962,7 +962,7 @@ namespace internal
               std::upper_bound(category_size.begin(), category_size.end(), i) -
                 category_size.begin() :
               0;
-          const std::array<unsigned int, 3> next{category_hp, max_index, i};
+          const std::array<unsigned int, 3> next{{category_hp, max_index, i}};
           if (batch_with_comm[i / n_lanes])
             batch_order_comm.emplace_back(next);
           else

